server/src/weaverbird/backends/mongo_translator/steps/__init__.py

```diff
@@ -0,0 +1,19 @@
+from typing import Any, Callable, Dict
+
+from weaverbird.backends.mongo_translator.steps.argmax import translate_argmax
+from weaverbird.backends.mongo_translator.steps.argmin import translate_argmin
+from weaverbird.backends.mongo_translator.steps.domain import translate_domain
+from weaverbird.backends.mongo_translator.steps.filter import translate_filter
+from weaverbird.backends.mongo_translator.steps.text import translate_text
+from weaverbird.pipeline.steps import BaseStep
+
+# I would like to have a better type for the callable, but I don't know how to do it.
+# each of this function take a particular step as input
+# so it is not possible to use `Dict[str, Callable[[BaseStep], list]]
```

Comment:
:'( it's because it says BaseStep is not compatible with each concrete step? wierd!

Reply:
yes. translate_filter does not take a `BaseStep` as an argument, but a `FilterStep` I am not sure how to express this in python, in java it would be something like `Function<? extends BaseStep, List>